x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_signal.test.ts

```diff
@@ -84,94 +87,25 @@ describe('buildSignal', () => {
       module: 'system',
     };
     const rule = getPartialRulesSchemaMock();
-    const signal = buildSignal(doc, rule);
+    const signal = {
+      ...buildSignal([doc], rule),
+      ...additionalSignalFields(doc),
+    };
     const expected: Signal = {
-      parent: {
-        rule: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
-        id: 'd5e8eb51-a6a0-456d-8a15-4b79bfec3d71',
-        type: 'event',
-        index: 'myFakeSignalIndex',
-        depth: 1,
-      },
-      ancestors: [
+      parents: [
         {
-          rule: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
           id: 'd5e8eb51-a6a0-456d-8a15-4b79bfec3d71',
           type: 'event',
           index: 'myFakeSignalIndex',
-          depth: 1,
+          depth: 0,
         },
       ],
-      original_time: '2020-04-20T21:27:45+0000',
-      original_event: {
-        action: 'socket_opened',
-        dataset: 'socket',
-        kind: 'event',
-        module: 'system',
-      },
-      status: 'open',
-      rule: {
-        created_by: 'elastic',
-        description: 'Detecting root and admin users',
-        enabled: true,
-        false_positives: [],
-        from: 'now-6m',
-        id: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
-        immutable: false,
-        index: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
-        interval: '5m',
-        risk_score: 50,
-        rule_id: 'rule-1',
-        language: 'kuery',
-        max_signals: 100,
-        name: 'Detect Root/Admin Users',
-        output_index: '.siem-signals',
-        query: 'user.name: root or user.name: admin',
-        references: ['http://www.example.com', 'https://ww.example.com'],
-        severity: 'high',
-        updated_by: 'elastic',
-        tags: ['some fake tag 1', 'some fake tag 2'],
-        to: 'now',
-        type: 'query',
-        note: '',
-        updated_at: signal.rule.updated_at,
-        created_at: signal.rule.created_at,
-      },
-    };
-    expect(signal).toEqual(expected);
-  });
-
-  test('it builds a signal as expected with original_event if is present and without internal tags in them', () => {
```

Comment:
This test was deleted and confused git...`buildSignal` is no longer responsible for removing internal tags. Test is replaced above with a similar test in `build_rule.test.ts`